persistence-jdbc/core/src/main/resources/play/reference-overrides.conf

```diff
@@ -27,3 +27,9 @@ jdbc-snapshot-store.slick {
 
 akka.persistence.journal.plugin = jdbc-journal
 akka.persistence.snapshot-store.plugin = jdbc-snapshot-store
+
+# To enable services to start when database is not available
+#
+# @deprecated: Remove in Lagom 1.5.x since this is the default
```

Comment:
Is that a way to annotate configurations/overrides to remove later?

Reply:
As a way to deprecate it looks good to me.

We will need a separate PR in `master` (not backportable) that removes this setting.